### ISSUE TITLE
CuBERT fine-tuning parameters

### ISSUE THREAD

**Issue by wenting-zhao** [google-research/google-research#734]
Hi there,

Thank you for the amazing work and for making the fine-tuning datasets online.  I was wondering if it would be possible to make cubert's fine-tuning parameters available to us?  Looking at the paper, it said,

> We pre-train CuBERT with the default configuration of the BERT Large model, one model per example length (128, 256, 512, and 1,024 subword tokens) with batch sizes of 8,192, 4,096, 2,048, and 1,024 respectively, and the default BERT learning rate of 1 × 10−4. Fine-tuned models also used the same batch sizes as for pre-training, and BERT’s default learning rate (5 × 10−5). For both, we gradually warm up the learning rate for the first 10 % of examples, which is BERT’s default value.

What do you mean by with batch sizes 8,192, 4,096, 2,048, and 1,024?  This seems not to match what I found on this page https://github.com/google-research/bert/blob/master/README.md.  Also, the paper said the evaluation is done on either V100 or P100.  Could you also specify the memory limit on those GPUs?

For the exception dataset, I can only achieve an accuracy of 75%, which is 4% away from what was reported, so I would greatly appreciate any help, as we find this work very interesting!  Thank you in advance!
